README.md

```diff
@@ -1,6 +1,25 @@
 # Equigy client
 
-A Java library with a client implementation to interact with the Tennet Equigy API.
+A Java library with a client implementation to interact with the TenneT Equigy API.
+
+## Example usage
+
+To obtain activated energy for a single EAN for a single date, see the following example:
+
+```java
+final OAuthClient oAuthClient = new OAuthClient();
+final ActivatedEnergyClient activatedEnergyClient = ActivatedEnergyClient.acceptance();
+
+final EquigyCredentials equigyCredentials = new EquigyCredentials();
+// Fill in the credentials for your organisation
+
+final OAuthToken oAuthToken = oAuthClient.retrieveToken(equigyCredentials);
+final List<ActivatedEnergy> activatedEnergies = activatedEnergyClient.getByQuery(
+        new EAN18("130539057492609625"),
+        LocalDate.of(2020, 12, 20),
+        oAuthToken
+);
+```
```

Comment:
Moeten we ook opgeven wie de maintainers zijn en wat te doen bij vragen/bugs/security breaches reporting?